UnityProject/Assets/Scripts/Core/Lifecycle/PlayerSpawn.cs

```diff
@@ -23,13 +24,26 @@ public class SpawnEventArgs : EventArgs { public GameObject player; }
 	/// <param name="joinedViewer">viewer who should control the player</param>
 	/// <param name="occupation">occupation to spawn as</param>
 	/// <param name="characterSettings">settings to use for the character</param>
+	/// <param name="showBanner">whether to show banner when spawned</param>
 	/// <returns>the game object of the spawned player</returns>
-	public static GameObject ServerSpawnPlayer(JoinedViewer joinedViewer, Occupation occupation, CharacterSettings characterSettings)
+	public static GameObject ServerSpawnPlayer(JoinedViewer joinedViewer, Occupation occupation, CharacterSettings characterSettings, bool showBanner = true)
 	{
 		NetworkConnection conn = joinedViewer.connectionToClient;
 
 		// TODO: add a nice cutscene/animation for the respawn transition
 		var newPlayer = ServerSpawnInternal(conn, occupation, characterSettings, null);
+
+		if (occupation != null && showBanner)
```

Comment:
at some point we need to reparent Occupation and Antagonist to be children of the same base class so we don't need duplicated code for stuff like this.